pkg/typesystem/typesystem_test.go

```diff
@@ -6070,624 +6070,103 @@ func TestPathExists(t *testing.T) {
 		pathTests []pathTest
 	}{
 		{
-			name: "userset_computed_userset",
```

Comment:
FYI The tests were moved to `language`. The tests I left are exclusively testin the code in `openfga`.